### PR TITLE
[AutoMM] Tutorial of finetuning SAM

### PR DESCRIPTION
**Pull request by zhiqiangdon**
*Issue #, if available:*

*Description of changes:*

- Finetuning SAM tutorial.
- Refactor all visualizers into one file.
- Small SAM fixs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/src/autogluon/multimodal/utils/semantic_segmentation_visualizer.py [hunk 1]
```diff
@@ -36,16 +39,7 @@ def show_mask(mask, ax):
                 continue
             show_mask(pred == class_id, plt.gca())
 
-        plt.axis("off")
+        # plt.axis("off")
```
[the remark this answers is not in the document]
Comment (zhiqiangdon) [reply multimodal/src/autogluon/multimodal/utils/semantic_segmentation_visualizer.py:42]: Do we need to turn off for both image and mask? I find having the axis on makes it easier to compare the segmentation results.

File: docs/tutorials/multimodal/index.md [hunk 2]
```diff
@@ -113,6 +113,20 @@ In the following, we decomposed the functionalities of AutoMM and prepared step-
 ::::
 
 
+<<<<<<< HEAD
```
Comment (tonyhoo) [inline docs/tutorials/multimodal/index.md:116]: Are we intended to keep this or it is leftover from conflict resolution?

Comment (zhiqiangdon) [reply docs/tutorials/multimodal/index.md]: Removed.

**Review by taoyang1122 — APPROVED** [review]
LGTM.

**Pull request merged**